src/index.ts

```diff
@@ -75,6 +76,23 @@ const plugin: JupyterFrontEndPlugin<void> = {
       tooltip: trans.__('Open a new folder')
     });
 
+    toolbarRegistry.registerFactory(
+      'FileBrowser', // Factory name
+      'OpenNewFolder',
+      (browser: FileBrowser) => openDirectoryButton
+    );
+
+    setToolbar(
+      widget,
+      createToolbarFactory(
+        toolbarRegistry,
+        settingRegistry!,
+        'FileBrowser',
+        '@jupyterlab/filebrowser-extension:widget',
```

Comment:
Maybe we should define our own toolbar items in the settings and change this to the following?

```suggestion
        'jupyterlab-filesystem-access:plugin',
```

Reply:
Given that we want to remove the "Open Launcher" button I guess it makes sense to do that indeed.